tools/ml-dataprep/queries.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Copyright line (see previous comment).

Reply:
Fixed in dc4430f316956c837b2ae237e4ea85a9489cb8fe